test/functional/config.js

```diff
@@ -44,14 +44,17 @@ export default async function({ readConfigFile }) {
 
     kbnTestServer: {
       ...commonConfig.get('kbnTestServer'),
-      serverArgs: [...commonConfig.get('kbnTestServer.serverArgs'), '--oss'],
+      serverArgs: [
+        ...commonConfig.get('kbnTestServer.serverArgs'),
+        '--oss',
+        '--telemetry.optIn=false',
```

Comment:
This is the new flag since `'telemetry:optIn'` was deprecated LONG time ago. both still work for backwards compatibilty.